pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -821,6 +853,10 @@ func (r *ReconcileVirtualMachineImport) validate(instance *v2vv1alpha1.VirtualMa
 		}
 		if valid, message := shouldFailWith(conditions); !valid {
 			logger.Info("Import blocked. " + message)
+
+			// Emit event vm import blocked:
+			r.recorder.Eventf(instance, corev1.EventTypeNormal, EventImportBlocked, "Virtual Machine import blocked: %s", message)
```

Comment:
please add vm name

Reply:
If the VM is blocked, that event will be emitted on every reconciliation until the cause for the block is fixed; I think we might need some throttling around recording of that event.

Reply:
Well, we can see following in the events of vm import:
`
  Normal  ImportBlocked  1s (x23 over 79s)  virtualmachineimport-controller  Virtual Machine import blocked: VM uses unsupported bios type: i440fx_sea_bios
`

and following in get events:
`
50s         Normal    ImportBlocked             virtualmachineimport/myvm                                    Virtual Machine import blocked: VM uses unsupported bios type: i440fx_sea_bios
`

I think it's fine, it's not flooded, you just see counter. Or I can check whether this event was already generated for this VM import and don't generate it if it was.

Reply:
It does not look that bad. I wonder if in case of mass import, when there is some issue with, for example, resource mapping for tens or hundreds of VMs the event rate would be problematic. My gut tells me that for a good event processing service thousands of events per second should not be a problem.